### DISCUSSION TITLE
ShadowDOM

### DISCUSSION THREAD

**Discussion by ArnaudVanHerck** [ghiscoding/aurelia-slickgrid#540] [Q&A]
Hi,

I was wondering if anyone have been able to get Aurelia-Slickgrid working with `@useShadowDOM` tag above the class name in Aurelia?
Whenever we've tried this the grid doesn't seem to find jQuery anymore and gives us errors in the console.
Is there something we're missing for this to work?

Kind regards,
Arnaud

**Comment by ghiscoding** [comment]
@ArnaudVanHerck there was this Example 29 that @zewa666 added with slots for Header & Footer, but apart from that I am not personally using ShadowDOM directly, not sure if @zewa666 can provide more help on the subject.

**Reply by ArnaudVanHerck** [reply]
Oh ok, I'll have to check out that example. Thanks for another quick reply. 😃

**Reply by zewa666** [reply]
Can you create a small repro @ArnaudVanHerck ?

**Reply by ArnaudVanHerck** [reply]
I'll see what I can throw together 😉

**Reply by ArnaudVanHerck** [reply]
Seems to be more difficult than I expected to reproduce this error in a new project, making me feel the error is in our code. 
Thank you all for taking you time answering!
